Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15648,6 +15648,11 @@ FloatPoint WebPageProxy::mainFrameScrollPosition() const
 
 #endif // PLATFORM(COCOA) && ENABLE(ASYNC_SCROLLING)
 
+void WebPageProxy::fetchSessionStorage(CompletionHandler<void(HashMap<WebCore::ClientOrigin, HashMap<String, String>>&&)>&& completionHandler)
+{
+    protectedWebsiteDataStore()->protectedNetworkProcess()->sendWithAsyncReply(Messages::NetworkProcess::FetchSessionStorage(sessionID(), identifier()), WTFMove(completionHandler));
```

Comment:
`if (RefPtr networkProcess = websiteDataStore().networkProcessIfExists())`
If network process does not exist yet, we don't need to launch one to fetch data

Reply:
Fixed!